rotkehlchen/exchanges/coinbasepro.py

```diff
@@ -388,6 +388,14 @@ def query_online_deposits_withdrawals(
         account_to_currency = self.create_or_return_account_to_currency_map()
         for entry in raw_movements:
             try:
+                # Check if the transaction has been canceled. If so it should be skipped
+                if 'cancel_code' in entry['details'].keys():
```

Comment:
Hmm I think your hypothesis has merit @yabirgb. I also can't test it since I did not get the error. Only the user who reported it can test it.

But looking at the list deposit or list withdrawals https://docs.pro.coinbase.com/#list-withdrawals api endpoint docs there I can't confirm a `'cancel_code'` key in that endpoint.

I believe it would be a much simpler approach to check if `'completed_at'`  exists in the object and if it does not exist, or if it's `None` then skip with the warning as you already do